core/src/token_info.rs

```diff
@@ -51,6 +51,11 @@ impl TokenBaseInfo {
         SYMBOL_OVERRIDES.get(&self.alias).unwrap_or(&self.alias)
     }
 
+    /// One unit of the token taking decimals into account, given in number of atoms.
+    pub fn unit_in_atoms(&self) -> f64 {
```

Comment:
ubernit: maybe base_unit instead of just unit (since technically an atom is also a unit)